src/pages/certificates/create-certificate/create-certificate.js

```diff
@@ -7,37 +10,82 @@ import {
   TextField,
   Tooltip
 } from '@material-ui/core';
-import { useStyles } from './create-certificate.styles';
-import { BackButton } from '../../../components/buttons';
-import materialUiConstants from '../../../configs/material-ui-constants';
-import { useCommonStyles } from '../../common.styles';
-import CheckBoxes from '../checkBoxes';
+
+import { LOCAL_STORAGE } from '../../../consts/local-storage';
+import { getFromLocalStorage } from '../../../services/local-storage.service';
+
+import {
+  setSnackBarMessage,
+  setSnackBarSeverity,
+  setSnackBarStatus
+} from '../../../redux/snackbar/snackbar.actions';
+import LoadingBar from '../../../components/loading-bar';
+
 import formRegExp from '../../../configs/form-regexp';
-import CertificatesTable from '../certificatesTable';
 import { loginErrorMessages } from '../../../configs/error-messages';
 import buttonTitles from '../../../configs/button-titles';
 import titles from '../../../configs/titles';
 import routes from '../../../configs/routes';
+import materialUiConstants from '../../../configs/material-ui-constants';
 
-const { certificatesTitles } = titles;
+import CheckBoxes from '../checkBoxes';
+import { BackButton } from '../../../components/buttons';
+import { useCommonStyles } from '../../common.styles';
+
+import { useStyles } from './create-certificate.styles';
+import CertificatesTable from '../certificatesTable';
+import { bulkGenerateCertificates } from '../operations/certificate.mutation';
+
+const { certificatesTitles, certificatesValueTitles } = titles;
 
 const CreateCertificate = () => {
+  const dispatch = useDispatch();
+
   const commonStyles = useCommonStyles();
   const styles = useStyles();
 
-  const initialCheckboxes = [
-    { checked: false, quantity: 1, name: certificatesTitles[500] },
-    { checked: false, quantity: 1, name: certificatesTitles[1000] },
-    { checked: false, quantity: 1, name: certificatesTitles[1500] }
-  ];
+  const arrValues = Object.keys(certificatesValueTitles);
+  const initialCheckboxes = arrValues.map((key) => ({
+    checked: false,
+    quantity: 1,
+    name: certificatesValueTitles[key],
+    value: Number(key)
+  }));
 
   const [checkBoxes, setCheckBoxes] = useState(initialCheckboxes);
-  const [date, setDate] = useState(new Date());
+  const [date, setDate] = useState();
```

Comment:
Are you still using this state somewhere?

Reply:
it is used only for rerendering the second Datepicker, if the first one was changed. I think that this Datepicker is useless, but we have it on mockups. I don't have any ideas how to do it without useState hook.
![test2](https://user-images.githubusercontent.com/24919819/152979158-9632b28c-0334-41ae-9407-4d9dd92fcaca.gif)